docs/posters-and-talks.rst

```diff
@@ -0,0 +1,34 @@
+Posters and Talks
+=================
+
+This page includes various talks and posters that the Dockstore team has given.
+
+
+
+2019
+----
+
+BOSC 2019
+^^^^^^^^^
+Dockstore 2.0: Enhancing a community platform for sharing cloud-agnostic research tools
+
+- :download:`Poster </_static/2019-BOSC_conference_poster.pdf>`
```

Comment:
Should include citation info via DOI to allow people to cite this presentation. 
This should be auto-generated on the F1000 site ala the other issue with links